client/src/components/LoginAndReg/Login.js

```diff
@@ -13,16 +13,20 @@ const Login = (props) => {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
   const [error, setError] = useState('')
-  const [success, setSuccess] = React.useState(false);
-  const [msg, setMsg] = React.useState('');
+  const [success, setSuccess] = useState(false);
+  const [msg, setMsg] = useState('');
   const { request_error } = useSelector(state => state.network);
 
   useEffect(() => {
     if (request_error && request_error.error) {
       setMsg(request_error.error)
       setError(true);
     }
-  }, [])
+    else {
+      setMsg('');
+      setError(false);
+    }
```

Comment:
This isn't needed. Reason being, when the component detects a change or remounts in general, the default state of msg and success is '' and false respectively.

Reply:
This is correct.  This change is not needed.  However, I'd like to discuss the other changes